examples/14-cancel-order.py

```diff
@@ -27,8 +29,9 @@ def main():
         #
         # See: https://docs.mollie.com/reference/v2/orders-api/cancel-order
         #
+        order_id = flask.request.args.get('order_id')
 
-        order = next(mollie_client.orders.list())
+        order = mollie_client.orders.get(order_id) if order_id else next(mollie_client.orders.list())
```

Comment:
Check the other examples. When you are cancelling a specific order, tell the user that you're doing this.